examples/simple_example/main.tf

```diff
@@ -46,7 +46,7 @@ locals {
 }
 
 module "healthcare" {
-  source  = "terraform-google-modules/healthcare/google"
```

Comment:
Was this autogenerated by `make generate_docs`?

Reply:
No. I was getting an error in this specific file when I defined the `google_healthcare_workspace` resource within the module. This was because it was not present in the remote version of `source  = "terraform-google-modules/healthcare/google"`. It was only present in my local version as I had just made those changes. So to support the local version (which might be the case when new resources are added), I changed it.

Reply:
The examples directory should be autogenerated only (https://github.com/terraform-google-modules/terraform-google-healthcare/blob/master/CONTRIBUTING.md#generating-documentation-for-inputs-and-outputs).

https://github.com/terraform-google-modules/terraform-google-healthcare/pull/33 is an example here.

Reply:
I have used `make generate_doc` in the latest commit. However it only generated a few lines in the README.md file in the root folder.

Reply:
It is quite weird to use local path for module source. Maybe one way is to submit this CL without the example change, then submit the example change using `source  = "terraform-google-modules/healthcare/google"` in a follow up CL. @imrannayer Please let me know if I'm missing anything here.